### ISSUE TITLE
Finalize boundaries

### ISSUE THREAD

**Issue by dabreegster** [nptscot/scheme-sketcher#4]
I'm using the boundaries described by https://github.com/nptscot/scheme-sketcher/blob/main/assets/README.md. Robin also mentioned 6 regions, but there are more than 6 here -- should we try larger areas?

Also, the boundaries are somewhat poor quality; https://nptscot.github.io/scheme-sketcher/sketch.html?boundary=City%20of%20Edinburgh looks like this near the edges:
![image](https://github.com/user-attachments/assets/cc3e892f-c605-4f23-80df-8c32de9b5913)
I can look for different 2023 LAD boundaries. If they're like the ones in England, there are variations that follow the coastline differently.

**Comment by Robinlovelace** [comment]
I can send you the region boundaries Dustin...

**Comment by Robinlovelace** [comment]
6 of them here: https://github.com/nptscot/inputdata/blob/main/boundaries/scottish_regions_2023.geojson

I think as with the other version 🏴󠁧󠁢󠁥󠁮󠁧󠁿 it could be good to have dual levels, with default being LAD level.

**Comment by dabreegster** [comment]
What's the original source for both of these files, aka, the date? I want to start from BFE to fix the shape (see https://geoportal.statistics.gov.uk/ for an explanation of the variations). I think you're using BGC right now.

I've found https://geoportal.statistics.gov.uk/datasets/3cba595dd06848f68879e3f2c3604f7e_0/explore for LADs from May 2024, and I also see May and December 2023.

I haven't found a source for regions yet.

**Comment by Robinlovelace** [comment]
The regions I guess are aggregations of the LAs, with the lookup between LA and region used as the basis for aggregation. Here's a list of Scottish regions that may in fact be better than ours (or may be the same, not checked): https://en.wikipedia.org/wiki/Subdivisions_of_Scotland#Transport

**Comment by Robinlovelace** [comment]
Update: original LAs dataset (it was BSC but any can be used): https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/Local_Authority_Districts_December_2023_Boundaries_UK_BSC/FeatureServer/0/query?outFields=*&where=1%3D1&f=geojson

Reproducible code to generate the regions from those in R: https://github.com/nptscot/npt/blob/b06988d3181dff0a8674cf23e578b3928b9f99f1/code/prep_admin_bounds.R#L27-L70

What's the rationale for starting from BFE? Maybe we should update on our side for consistency if there are good reasons.

**Comment by Robinlovelace** [comment]
And in terms of how I found this, see here in case of use/interest for navigating the codebase: https://github.com/search?q=repo%3Anptscot%2Fnpt%20regions&type=code

**Comment by Robinlovelace** [comment]
The boundary shape in the original post does look a bit weird admittedly, a good reason for changing!

**Comment by dabreegster** [comment]
Not sure I'll have time to make nicer boundaries that fit the coastline (simplifying the high-res ones while preserving the coastline isn't clear), so for now, I've used the region boundaries as provided. The route snapper files are larger and thus slower to load in big areas, but they're not too bad. I'm still waiting on the Highland region to generate; I think the large number of islands in the multipolygon is breaking some assumptions in my clipping routine. But otherwise, ready to go at https://nptscot.github.io/scheme-sketcher/

**Comment by Robinlovelace** [comment]
Cool!

**Comment by Robinlovelace** [comment]
![image](https://github.com/user-attachments/assets/e98b7111-592f-4aec-ad85-76ea8e0197ef)

**Comment by Robinlovelace** [comment]
Impressive. Great work Dustin. People have been on leave but look forward to setting-up a meeting with @anguscalder and co soon, and of course using the data generated!

**Comment by Robinlovelace** [comment]
Updated regions coming your way

**Comment by Robinlovelace** [comment]
Can you update the regions based on this (same file name, new contents): https://github.com/nptscot/npt/releases/download/boundaries-2024/la_regions_scotland_bfe_simplified_2023.geojson

**Comment by Robinlovelace** [comment]
Then we can close this issue.

**Comment by Robinlovelace** [comment]
Closing in favour of this: https://github.com/nptscot/npw/issues/58
